requirements/base.txt

```diff
@@ -119,6 +119,8 @@ flask-caching==2.3.1
     # via apache-superset (pyproject.toml)
 flask-compress==1.17
     # via apache-superset (pyproject.toml)
+flask-cors==4.0.2
```

Comment:
oh, wondering if you added this manually? This file is supposed to be compiled by `uv pip compile` via the script at https://github.com/apache/superset/blob/master/scripts/uv-pip-compile.sh .  I think the proper way is to alter https://github.com/apache/superset/blob/master/requirements/base.in#L19 to add a line referencing to `.[cors]`, and then running the script...

Reply:
I'm trying to clarify how it should be done here -> https://github.com/apache/superset/pull/33714

In you case I think adding a line for `.[cors]` in `base.in` makes most sense. Either that or promoting the one dep in cors to the core set of dependencies in `pyproject.toml`